molgenis-api-identities/src/main/java/org/molgenis/api/identities/GroupRestController.java

```diff
@@ -229,6 +230,73 @@ public void updateMember(
     groupService.updateMemberRole(group, member, newRole);
   }
 
+  @PostMapping(ROLE_EXTEND_END_POINT)
```

Comment:
How about removing the POST endpoint and keep the PUT endpoint (to add an item to the includes list) and DELETE endpoint (to remove an item from the includes list)? The PUT and delete endpoints should validate whether they are adding/removing group roles (and not other roles).